core/src/com/group66/game/screens/MainMenuScreen.java

```diff
@@ -6,84 +6,132 @@
 import com.badlogic.gdx.graphics.Texture;
 import com.group66.game.BustaMove;
 
-public class MainMenuScreen implements Screen{
+/**
+ * A Class for the MainMenuScreen of the game.
+ */
+public class MainMenuScreen implements Screen {
 
+	/** The Constant PLAY_BUTTON_WIDTH. */
```

Comment:
The whole menu screen needs an overhaul, I just left it in there because the menu screen will get its own branch for development